manifests/init.pp

```diff
@@ -63,8 +63,8 @@
 
   Optional[String] $repo_url = undef,
   String  $repo_name = 'influxdb2',
-  String  $version = '2.1.1',
-  Variant[String,Boolean[false]] $archive_source = 'https://dl.influxdata.com/influxdb/releases/influxdb2-2.1.1-linux-amd64.tar.gz',
+  String  $version = '2.6.1',
+  Variant[String,Boolean[false]] $archive_source = 'https://dl.influxdata.com/influxdb/releases/influxdb2-2.6.1-linux-amd64.tar.gz',
```

Comment:
Is that a breaking change? Will it upgrade existing installation to 2.6.1?

Reply:
Hopefully not, the [changelog](https://docs.influxdata.com/influxdb/v2.6/reference/release-notes/influxdb/#v261-2022-12-29) just mentions bugfixes and security updates.  I upgraded my test instance and don't see any obvious issues with InfluxDB or Grafana panels.